fields/types/password/PasswordType.js

```diff
@@ -133,6 +150,17 @@ password.prototype.validateInput = function (data, callback) {
 		detail = 'passwords must match';
 	}
 	// TODO: we could support a password complexity option (or regexp) here
+	if (min && typeof passwordValue === 'string') {
+		result = passwordValue.length > min;
+	}
+	for (var prop in complexity) {
+		if (complexity[prop] && typeof passwordValue === 'string') {
+			result = (regexChunk[prop]).test(passwordValue);
```

Comment:
Oh right. If we're validating options, that should happen in the constructor and throw there (so it shows the error when keystone starts, not when validation is called while the app is running)